src/debuginfo.cpp

```diff
@@ -1515,8 +1515,10 @@ static DW_EH_PE parseCIE(const uint8_t *Addr, const uint8_t *End)
     // Assume there's no EH Data field, which exist when the augmentation
     // string has "eh" in it.
     p += augmentation_len + 1;
-    // Code Alignment Factor (1 byte) should always be 1
-    assert(*p == 1);
+    // Code Alignment Factor (1 byte)
+    // should always be 1 on x86, 4 on PPC, etc.
+    // (used for DW_CFA_advance_loc / not used here)
+    //assert(*p == 1);
```

Comment:
what's the correct check? the only reference seems to be some random guys blog at the url above. I'm surprised it's not 4 on ARM also, but perhaps due to thumb mode accommodations it was 1 there too? If we were to need to use this parameter, then presumably we would actually use it's value, instead of pre-asserting something about the environment